.coderabbit.yaml

```diff
@@ -1,3 +1,49 @@
+language: "en-US"
+tone_instructions: ""
+early_access: true
+enable_free_tier: true
+
 reviews:
-  review_status: false
-  high_level_summary: false
+  high_level_summary: true
+  high_level_summary_placeholder: "@coderabbitai summary"
+  review_status: true
```

Comment:
It looks like this setting changed from false to true in the recent update. The bot has started leaving comments like "Review Skipped" on PRs it created. If it's unnecessary, we might consider reverting it to false in the future.